esmvaltool/diag_scripts/kcs/local_resampling.py

```diff
@@ -0,0 +1,465 @@
+"""Resample the target model for the selected time periods."""
+import logging
+from itertools import product
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import xarray as xr
+from tqdm import tqdm
```

Comment:
Please add this new dependency to setup.py and package/meta.yml, but only if really needed.

Reply:
Decided to remove it.